src/main/scala/io/dronekit/cloud/AWSWrapper.scala

```diff
@@ -170,7 +170,7 @@ class AWSWrapper(S3Client: AmazonS3Client = S3.client)
   }
 
   def streamUpload(s3url: S3URL): Sink[ByteString, Future[CompleteMultipartUploadResult]] = {
-    multipartUploadTransform(s3url).fold(Seq.empty[UploadPartResult])(_ :+ _).to(Sink.head)
+    multipartUploadTransform(s3url).fold(Seq.empty[UploadPartResult])(_ :+ _).to(Sink.ignore)
```

Comment:
[`def ignore: Sink[Any, Future[Done]]`](http://doc.akka.io/api/akka/2.4/index.html#akka.stream.scaladsl.Sink$@ignore:akka.stream.scaladsl.Sink[Any,scala.concurrent.Future[akka.Done]])
`A Sink that will consume the stream and discard the elements.`

You're using `Sink.ignore` without the fold for the same purpose in streamInsertIntoBucket.